WcaOnRails/app/views/competitions/_index_competitions_list.html.erb

```diff
@@ -50,4 +50,45 @@
       });
     });
   </script>
+<% else %>
+  <% if current_user&.can_see_admin_competitions? %>
+    <% if competitions.empty? %>
+      <div class="col-md-12">
+        <%= alert :warning do
+          unless params[:event_ids].empty?
+            "We didn't find any competitions with those #{ pluralize(params[:event_ids].count, "event" )}! Try searching for fewer events."
+          else
+            "No competitions found."
+          end
+        end %>
```

Comment:
1. I think that `<div class="col-md-12">` is missing closing `div`. The same goes for competitions list.
2. This is exactly the same as for competitions list, it's worth making `no_competitions_alert` - either partial for the whole `div` or a helper method like returning the result of `alert :warning ...`.